test/extensions/filters/http/header_to_metadata/header_to_metadata_filter_test.cc

```diff
@@ -443,6 +443,55 @@ TEST_F(HeaderToMetadataTest, PerRouteEmtpyRules) {
   EXPECT_THROW(std::make_shared<Config>(config_proto, true), EnvoyException);
 }
 
+/**
+ * Invalid empty header or cookie should be rejected.
+ */
```

Comment:
nit:

```
// Invalid empty header or cookie should be rejected.
```

Reply:
@rgs1 the only reason I used multi-line comment is because every other test in that file had that and I wanted to keep it consistent. should I change all of them?

Reply:
Ah gotcha, it's good then.